usaspending_api/recipient/delta_models/recipient_profile.py

```diff
@@ -1,26 +1,330 @@
+            ELSE CONCAT('name-', COALESCE(fpds.awardee_or_recipient_legal, fabs.awardee_or_recipient_legal, '')) END
+        )), '^(\.{{8}})(\.{{4}})(\.{{4}})(\.{{4}})(\.{{12}})$', '\$1-\$2-\$3-\$4-\$5') AS recipient_hash,
+        COALESCE(fpds.awardee_or_recipient_uniqu, fabs.awardee_or_recipient_uniqu) AS recipient_unique_id,
+        COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide) AS parent_recipient_unique_id,
+        COALESCE(fpds.awardee_or_recipient_uei, fabs.uei) AS uei,
+        COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei) AS parent_uei,
+        CASE
+            WHEN tn.type IN ('A', 'B', 'C', 'D')      THEN 'contract'
+            WHEN tn.type IN ('02', '03', '04', '05')  THEN 'grant'
+            WHEN tn.type IN ('06', '10')              THEN 'direct payment'
+            WHEN tn.type IN ('07', '08')              THEN 'loans'
+            WHEN tn.type IN ('09', '11')              THEN 'other'     -- collapsing insurance into other
+            WHEN tn.type LIKE 'IDV%'                  THEN 'contract'  -- collapsing idv into contract
+            ELSE NULL
+        END AS award_category,
+        CASE
+            WHEN COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei) IS NOT NULL THEN 'C'
+            ELSE 'R' END AS recipient_level,
+        tn.action_date,
+        CAST(COALESCE(CASE
+            WHEN tn.type IN('07','08') THEN tn.original_loan_subsidy_cost
+            ELSE tn.federal_action_obligation
+        END, 0) AS NUMERIC(23, 2)) AS generated_pragmatic_obligation
+    FROM
+        raw.transaction_normalized as tn
+    LEFT OUTER JOIN raw.transaction_fpds as fpds ON tn.id = fpds.transaction_id
+    LEFT OUTER JOIN raw.transaction_fabs as fabs ON tn.id = fabs.transaction_id
+    WHERE
+        tn.action_date >= '2007-10-01'
+        AND tn.type IS NOT NULL
+    );""",
+    #
+    # --------------------------------------------------------------------------------
+    # -- Step 2, Create the new table and populate with 100% of combinations
+    # --------------------------------------------------------------------------------
+    f"""
+        INSERT INTO {{DESTINATION_DATABASE}}.{{DESTINATION_TABLE}} (
+            SELECT
+                'P' as recipient_level,
+                recipient_hash AS recipient_hash,
+                duns AS recipient_unique_id,
+                legal_business_name AS recipient_name,
+                ARRAY() AS recipient_affiliations,
+                0.00 AS last_12_months,
+                0 AS id,
+                0.00 AS last_12_contracts,
+                0.00 AS last_12_grants,
+                0.00 AS last_12_direct_payments,
+                0.00 AS last_12_loans,
+                0.00 AS last_12_other,
+                0 AS last_12_months_count,
+                ARRAY() AS award_types,
+                uei AS uei,
+                parent_uei AS parent_uei
+            FROM
+                raw.recipient_lookup
```

Comment:
Changing this to `rpt.recipient_lookup` depends on getting https://github.com/fedspendingtransparency/usaspending-api/pull/3541 merged in, otherwise changing this will cause errors.